mylyn.github/org.eclipse.mylyn.github.core/META-INF/MANIFEST.MF

```diff
@@ -11,8 +11,8 @@ Export-Package: org.eclipse.mylyn.internal.github.core;version="6.2.0";x-friends
  org.eclipse.mylyn.internal.github.core.gist;version="6.2.0";x-friends:="org.eclipse.mylyn.github.ui",
  org.eclipse.mylyn.internal.github.core.issue;version="6.2.0";x-friends:="org.eclipse.mylyn.github.ui",
  org.eclipse.mylyn.internal.github.core.pr;version="6.2.0";x-friends:="org.eclipse.mylyn.github.ui"
-Require-Bundle: org.eclipse.mylyn.tasks.core;bundle-version="[3.26.0,4.0.0)",
- org.eclipse.mylyn.commons.net;bundle-version="[3.26.0,4.0.0)",
+Require-Bundle: org.eclipse.mylyn.tasks.core;bundle-version="[3.26.1,4.0.0)",
```

Comment:
I think we need to change the Bundle-Version to 6.2.1.qualifier for all Mylyn GitHub Artifacts